src/DocumentPicker.Plugin/Shared/DocumentConverterService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using PdfToSvg;
+using Xamarin.Essentials;
+
+namespace DocumentPicker.Plugin.Shared
+{
+    public class DocumentConverterService : IDocumentConverterService
+    {
+        public string ConvertPdfToSvgString(DocumentResult documentResult)
+        {
+            using (var doc =  PdfDocument.Open(documentResult.FullPath))
+            {
+                return doc.Pages[0].ToSvgString();
+            }
+        }
+
+        public void ConvertPdfToSvg(DocumentResult documentResult, string outputSource)
```

Comment:
please use the async variant!
and pass a cancellationtoken (`CancellationToken cancellationToken = default`)